packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -28,4 +29,18 @@ export interface ISemanticColors {
   inputBackgroundSelected?: string;
   inputBackgroundSelectedHovered?: string;
   inputForegroundSelected?: string;
+  inputFocusBorderAlt?: string;
+
+  // Menus, popups, etc
+  menuItemBackgroundHovered?: string;
```

Comment:
I think each of these should have a simple comment explaining what scenario its used in.